# HG changeset patch
# User Matt Johnston <matt@ucc.asn.au>
# Date 1376318460 -28800
# Node ID bcfa7f77953197d1e2a43b5770af3bebc503e0dd

Remove accidental one second sleep leftover from debugging

diff -r 028fa77f952f -r bcfa7f779531 svr-chansession.c
--- a/svr-chansession.c	Mon Jul 08 22:43:56 2013 +0800
+++ b/svr-chansession.c	Mon Aug 12 22:41:00 2013 +0800
@@ -699,8 +699,6 @@
 	ses.maxfd = MAX(ses.maxfd, channel->readfd);
 	ses.maxfd = MAX(ses.maxfd, channel->errfd);
 
-	sleep(1);
-
 	addchildpid(chansess, chansess->pid);
 
 	if (svr_ses.lastexit.exitpid != -1) {

